runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultConfig.java

```diff
@@ -40,4 +40,14 @@ default boolean showTimer()
 	{
 		return true;
 	}
+	
+	@ConfigItem(
+			keyName = "waveTimes",
```

Comment:
Fix the indentation (https://github.com/runelite/runelite/wiki/Code-Conventions just download this xml and import to Intellij)